app/services/instrument_offline_reservation_canceler.rb

```diff
@@ -18,7 +18,7 @@ def admin_user
 
   def cancel_reservation(reservation)
     reservation.order_detail.cancel_reservation(admin_user, admin: true)
-    reservation.update_attribute(:canceled_reason, "The instrument was offline")
+    reservation.order_detail.update_attributes(canceled_reason: "The instrument was offline")
```

Comment:
I wonder if now `canceled_reason` should be an optional argument in `OrderDetail#cancel_reservation`.